### PR TITLE
ISPN-14092 Cache Configuration Wizard Direct Link

### PR DESCRIPTION
**Pull request by dpanshug**
https://issues.redhat.com/browse/ISPN-14092

### CODE REVIEW

File: src/app/CacheManagers/CacheTableDisplay.tsx [hunk 1]
```diff
@@ -583,20 +583,21 @@ const CacheTableDisplay = (props: {
   };
 
   const buildCreateCacheButton = () => {
-    if (!ConsoleServices.security().hasConsoleACL(ConsoleACL.CREATE, connectedUser)) {
-      return '';
-    }
+
+    // Will remove comment when the feature is ready
```
Comment (karesti) [inline src/app/CacheManagers/CacheTableDisplay.tsx:587]: remove comment

File: src/app/Caches/Create/CreateCacheWizard.tsx [hunk 2]
```diff
@@ -229,11 +234,27 @@ const CreateCacheWizard = (props) => {
       let activeButton = true;
       switch (activeStepId) {
         case 1: activeButton = configuration.start.valid; break;
-        case 2: activeButton = true; break;
+        case 2:
+          if(ConsoleServices.security().hasConsoleACL(ConsoleACL.CREATE, connectedUser)){
```
Comment (karesti) [inline src/app/Caches/Create/CreateCacheWizard.tsx:238]: active button is already init to true. check if the user has not the right and set it to false instead and break

File: src/app/Caches/Create/CreateCacheWizard.tsx [hunk 3]
```diff
@@ -229,11 +234,27 @@ const CreateCacheWizard = (props) => {
...
+          else{
+            activeButton = false;
+            break;
+          }
         case 3: activeButton = configuration.basic.valid; break;
         case 4: activeButton = validFeatures(configuration); break;
         case 5: activeButton = configuration.advanced.valid; break;
-        case 6: activeButton = true; break
+        case 6:
+          if(ConsoleServices.security().hasConsoleACL(ConsoleACL.CREATE, connectedUser)){
```
Comment (karesti) [inline src/app/Caches/Create/CreateCacheWizard.tsx:250]: same code as above

**Review by karesti — CHANGES_REQUESTED** [review]
squash commits please

**Comment by karesti** [comment]
@dpanshug some comments to be addressed

**Comment by dpanshug** [comment]
> @dpanshug some comments to be addressed

Done :+1:

**Pull request merged**